packages/wix-ui-icons-common/stories/wix-ui-tpa/index.story.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import {
+  header,
+  tabs,
+  tab,
+  description,
+  importExample,
+  title,
+  divider,
+} from "wix-storybook-utils/Sections";
+import IconsExample from "../components/icons-example";
+import * as iconComponents from "../../src/wix-ui-tpa/dist";
+import API_Table from "../APITable";
+import WixUiTpaCategoryList from './WixUiTpaCategoryList'
```

Comment:
The file name is 
```suggestion
import WixUiTpaCategoryList from './WixTpaCategoryList'
```

So we'll need to either change the file name or the import